bids/modeling/model_spec.py

```diff
@@ -88,6 +88,9 @@ def __init__(self, terms=None, X=None, Z=None, groups=None, sigma=None,
         if Z is not None:
             self.build_variance_components(Z, groups, sigma)
 
+    def __repr__(self):
+        return f"<{self.__class__.__name__}{self.X.columns.tolist()}'>"
```

Comment:
`X` is a property that generates a dataframe. Instead, let's use what it uses to generate columns:

```suggestion
        return f"<{self.__class__.__name__}{[term.name for term in self.fixed_terms]}'>"
```